src/main/scala/org/broadinstitute/dsde/workbench/sam/api/AdminRoutes.scala

```diff
@@ -109,46 +109,59 @@ trait AdminRoutes extends SecurityDirectives with SamRequestContextDirectives wi
           }
-        } ~
-          pathPrefix(Segment / "memberEmails" / Segment) { case (policyName, userEmail) =>
-            val policyId = FullyQualifiedPolicyId(resource, AccessPolicyName(policyName))
+        }
+      } ~
+        pathPrefix("policies") {
+          withNonAdminResourceType(ResourceTypeName(resourceTypeName)) { resourceType =>
+            val resource = FullyQualifiedResourceId(resourceType.name, ResourceId(resourceId))
             pathEndOrSingleSlash {
-              withSubject(WorkbenchEmail(userEmail), samRequestContext) { subject =>
-                put {
-                  requireAdminResourceAction(adminAddMember, resourceType, user, samRequestContext) {
-                    complete {
-                      resourceService
-                        .addSubjectToPolicy(policyId, subject, samRequestContext)
-                        .as(NoContent)
-                    }
+              get {
+                requireAdminResourceAction(adminReadPolicies, resourceType, user, samRequestContext) {
+                  complete {
+                    resourceService
+                      .listResourcePolicies(resource, samRequestContext)
+                      .map(response => OK -> response.toSet)
                   }
-                } ~
-                  delete {
-                    requireAdminResourceAction(adminRemoveMember, resourceType, user, samRequestContext) {
-                      complete {
-                        resourceService
-                          .removeSubjectFromPolicy(policyId, subject, samRequestContext)
-                          .as(NoContent)
+                }
+              }
+            } ~
+              pathPrefix(Segment / "memberEmails" / Segment) { case (policyName, userEmail) =>
+                val policyId = FullyQualifiedPolicyId(resource, AccessPolicyName(policyName))
+                pathEndOrSingleSlash {
+                  withSubject(WorkbenchEmail(userEmail), samRequestContext) { subject =>
+                    put {
+                      requireAdminResourceAction(adminAddMember, resourceType, user, samRequestContext) {
+                        complete {
+                          resourceService
+                            .addSubjectToPolicy(policyId, subject, samRequestContext)
+                            .as(NoContent)
+                        }
+                      }
+                    } ~
+                      delete {
+                        requireAdminResourceAction(adminRemoveMember, resourceType, user, samRequestContext) {
+                          complete {
+                            resourceService
+                              .removeSubjectFromPolicy(policyId, subject, samRequestContext)
+                              .as(NoContent)
+                          }
+                        }
                       }
-                    }
                   }
+                }
               }
-            }
           }
-      }
```

Comment:
There was just a `PathPrefix("resources" / Segment / Segment / "policies")` here and now we have a new `PathPrefix("resources" / Segment / Segment)` delete endpoint and then the old one is nested inside of that one.